### DISCUSSION TITLE
Welcome to DeepSqueak Discussions!

### DISCUSSION THREAD

**Discussion by DrCoffey** [DrCoffey/DeepSqueak#118] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ande4808** [comment]
Hi, I am new to deepsqueak, but a former lab member has used it. However, she is no longer here, so I am mining to figure out how she generated her data. I was able to launch the program through matlab, but the neural network she used is not available in the dropdown. I would like to know how to get "Short Rat Call_Network_V2.mat." If you could direct me to the downloadable file I would so greatly appreciate it!

**Reply by ande4808** [reply]
In fact, all files in the neural network folder are missing so I am unable to generate any data from my audio files.

**Reply by Mansi-Purohit** [reply]
Are you looking for the standard networks that come with DeepSqueak or was this a new network trained by the former lab member?

**Reply by DrCoffey** [reply]
Hey @ande4808 sorry for the very slow reply. The old neural networks aren't compatible with the new version of DeepSqueak. I would recommend just using the new version 3.0 networks, they tend to work a bit better than the old ones. You can also download the final stable release of DeepSqueak v2.6 here: https://github.com/DrCoffey/DeepSqueak/releases/tag/V2.6.2

That has the old networks you are looking for.

**Comment by ande4808** [comment]
Thank you very much for your response! We have all but given up as our
undergrad was the spearhead for this project and has since left the lab. I
will try this new version with the hopes that the network for "short rat
call_network_V2.mat" is there. I appreciate your help.

Sincerely,

Kate Root
***@***.***
University of Minnesota Duluth-Medical School
Biomedical Sciences
Dr. Jean Regal

**Comment by ande4808** [comment]
Hello,
I was successful in opening the older version of DeepSqueak to utilize the
older neural networks. Now, I've run into a new problem! How do I open a
saved session in DeepSqueak? I have all the files for the saved sessions,
but I'm not sure how to actually open them again and view what was
previously saved. Can you give me some step by step instructions on how to
open those files?
Thank you for your time!
Kate Root
***@***.***

**Comment by yichiz123** [comment]
Hello! Thanks for creating such a wonderful program. I have been using it for the past couple of weeks and just have a quick question about syntax analysis plots: Is it possible to change the order of the nodes so that the call categories line up across plots? Right now, the syntax analysis plot switches two of my call categories for male and female mice as I'm trying to compare across 4 groups. Any assistance would be much appreciated :)

**Comment by DrCoffey** [comment]
Hey @yichiz123 I haven't used that feature in a while, but if you are comfortable with coding you can pause and debug in the  SyntaxAnalysis_callback.m file. You might be able to sort the labels before plotting to keep them in the same order?

Maybe add  cats=sort(cats,'descend'); to line 68. 

Sorry I don't have time to play with it right now.

**Comment by ande4808** [comment]
How do I open a saved session in DeepSqueak? I have all the files for the saved sessions, but I'm not sure how to actually open them again and view what was previously saved. Can you give me some step by step instructions on how to open those files?
Thank you for your time!
Kate Root

**Reply by DrCoffey** [reply]
Just select the saved detection file in the dropdown menu and then hit "load calls"

If they aren't in the default folder, first hit File-Select Detection Folder  and choose the correct folder.

**Comment by ande4808** [comment]
Oh yes, thank you! I see now. I appreciate you replying to my message.

**Comment by yichiz123** [comment]
@DrCoffey, Thank you so much! I just did sort(cats) and it worked :)

On Tue, Jun 7, 2022 at 2:49 PM DrCoffey ***@***.***> wrote:

> Hey @yichiz123 <https://github.com/yichiz123> I haven't used that feature
> in a while, but if you are comfortable with coding you can pause and debug
> in the SyntaxAnalysis_callback.m file. You might be able to sort the labels
> before plotting to keep them in the same order?
>
> Maybe add cats=sort(cats,'descend'); to line 68.
>
> Sorry I don't have time to play with it right now.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DrCoffey/DeepSqueak/discussions/118#discussioncomment-2900233>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ5B3NYUJFEBRDWP7M3BQ63VN6KSVANCNFSM5BFDKBTQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Yichi Zhang

**Comment by ande4808** [comment]
Hi,
Below is an image of one of the rat pup files that I'm trying to analyze in DeepSqueak. Why are the calls "stacked" on top of each other? Could that be an echo picked up by the microphone I used for USV detections? Are the calls in the higher frequency range not true calls? 

Thanks!

![DeepSqueak 7_25_2022 4_34_04 PM](https://user-images.githubusercontent.com/97688378/180879127-a1350605-6328-4cff-8b6d-9893e25195ba.png)
